firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java

```diff
@@ -351,6 +355,16 @@ private void addSequenceNumber() {
     }
   }
 
+  private void addUpdateTime() {
+    if (!tableContainsColumn("remote_documents", "snapshot_version_seconds")) {
```

Comment:
Since there are multiple versions of a document, it'd probably be better to follow the name used on the backend and call this the `update_time_seconds` -- that way it's clear what the correspondence is with the proto fields.

Reply:
Done. Future me will thank you for this :)